cfgmgr/vlanmgr.h

```diff
@@ -31,6 +34,10 @@ class VlanMgr : public Orch
 
     bool addHostVlan(int vlan_id);
     bool removeHostVlan(int vlan_id);
+
+	bool updateVlanStatusInit(vector<FieldValueTuple> &fvVector, const int vlan_id);
```

Comment:
This doesn't seem to be aligned

Reply:
Thanks，this is a mistake and will be fixed